fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -790,6 +791,10 @@ rollupRenameClause
     : RENAME ROLLUP rollupName=identifier newRollupName=identifier
     ;
 
+compactionClause
```

Comment:
I think `compact` is enough. The user should not take base/cumulative into consideration.

Reply:
base/cumulative is optional

Reply:
I think we can auto-determine to do cumulative compaction or base compaction.